customization/dark-theme.md

```diff
@@ -120,7 +120,7 @@ The advantage of creating a dark themed Template VM is, that each AppVM which is
 
 1. Start VM
 
-    **Note:** In case of App VM start the Template on which the AppVM is based on.
+    **Note:** In the case of App VM start the Template on which the AppVM is based on.
```

Comment:
"In case of App VM" sounds even more weird :)

Probably should be "If the VM is an AppVM, start the template on which the AppVM is based on"